data/pages-content/pyladies-workshop.md

```diff
@@ -0,0 +1,23 @@
+# Beginners' Day - PyLadies Snake Workshop
+## About PyLadies
+PyLadies is an international mentorship group with focus on helping more women become active participants and leaders in the Python open-source community.
+Our mission is to promote, educate and advance a diverse Python community through outreach, education, conferences, events and social gatherings.
+
+## About the Workshop 
+Are you a member of an under-represented group interested in programming but unsure where to begin? Don't worry, we have the perfect workshop for you, designed to be welcoming and accessible even if you don't have any prior programming experience!
+
+With the support of experienced mentors, you'll create your very own version of the classic Snake video game, reminiscent of those on older Nokia mobile phones. You'll get hands-on experience writing code, learning fundamental programming concepts, and feeling the satisfaction of a computer finally obeying your commands.
+
+If the project seems impractical, fear not! The main goal is to teach you basic programming principles that you can later apply to various fields, such as web application development, data analysis, machine learning, robotics, or system tool design.
+
+This workshop is specifically tailored for total beginners. There's no need for prior programming knowledge; all you need is a computer and basic skills to use it (e.g., opening/saving files, installing programs).
+
+The workshop will be conducted in English and is completely free. However, space is limited, so be sure to register through [this link](https://docs.google.com/forms/d/1r-8gQL1pMGJp45-jbVyEfJUcct0-RpbHUxEIo-hqReg/edit).
```

Comment:
How about using the same button style as the Trans*code event? (Link is still TBD)

```suggestion
The workshop will be conducted in English and is completely free. However, space is limited, so be sure to register using the form below.
<div style={{textAlign: "center", marginBottom: 20}}>
<ButtonLink href="https://docs.google.com/forms/d/1r-8gQL1pMGJp45-jbVyEfJUcct0-RpbHUxEIo-hqReg/edit"> Register your interest now! </ButtonLink>
</div>
```
(I haven't rendered it, the code is borrowed from the `Trans*Code` page).

Reply:
Great idea, I've committed it 🙂